portality/formcontext/formcontext.py

```diff
@@ -1052,7 +1060,7 @@ def finalise(self):
         # inform publisher if this was set to 'in progress' from 'pending'
         if self.source.application_status == constants.APPLICATION_STATUS_PENDING and self.target.application_status == constants.APPLICATION_STATUS_IN_PROGRESS:
             if app.config.get("ENABLE_PUBLISHER_EMAIL", False):
-                alerts = emails.send_publisher_inprogress_email(self.target)
+                alerts = emails.send_publisher_update_request_inprogress_email(self.target)
```

Comment:
I think that we need to introduce the ability to differentiate between an update request and a regular application here.  You can see this done elsewhere in this file by looking for the distinguishing mark:

```
is_update_request = self.target.current_journal is not None
```